client/my-sites/importer/controller.js

```diff
@@ -34,8 +34,8 @@ export function importSite( context, next ) {
 
 	context.store.dispatch(
 		setImportOriginSiteDetails( {
-			engine: get( context, 'state.engine' ),
-			siteUrl: decodeURIComponentIfValid( get( context, 'state.siteUrl' ) ),
+			siteEngine: get( context, 'state.engine' ),
```

Comment:
That's a good point--we use `importSiteEngine` as the signup dependency key, because that isn't nested, it's directly under `state.signup.dependencyStore`. With the importer-nux state, it's already clear that's information about the site being imported, so I've changed it back to `siteUrl`.

importer-nux state could use some work to make it clearer what it's for, but that's outside the scope of this PR.